recipes/rdkit/bld.bat

```diff
@@ -0,0 +1,51 @@
+rem surely there's a better way than this
+if "%PY_VER%"=="2.7" (
+	set PYTHON_LIBRARY=python27.lib
+	set AVALON=ON
+) else if  "%PY_VER%"=="3.4" (
+	set PYTHON_LIBRARY=python34.lib
+	set AVALON=OFF
+) else if  "%PY_VER%"=="3.5" (
+	set PYTHON_LIBRARY=python35.lib
+	set AVALON=OFF
+) else if  "%PY_VER%"=="3.6" (
+	set PYTHON_LIBRARY=python36.lib
+	set AVALON=OFF
+) else (
+	echo "Unexpected version of python"
+	exit 1
+)
+
+where jom 2> NUL
+if %ERRORLEVEL% equ 0 (
+  set MAKE_CMD=jom -j%CPU_COUNT%
+) else (
+  set MAKE_CMD=nmake
+)
+
+cmake ^
+    -G "NMake Makefiles" ^
+    -D RDK_INSTALL_INTREE=OFF ^
+    -D RDK_BUILD_INCHI_SUPPORT=ON ^
+    -D RDK_BUILD_AVALON_SUPPORT=%AVALON% ^
+    -D RDK_USE_FLEXBISON=OFF ^
+    -D Python_ADDITIONAL_VERSIONS=${PY_VER} ^
+    -D PYTHON_EXECUTABLE="%PYTHON%" ^
+    -D PYTHON_INCLUDE_DIR="%PREFIX%\include" ^
```

Comment:
Copied from before due to diff changes causing it to be hidden.

Not required, but as a helpful tip. Sometimes CMake on Windows needs to be told that some variables are `PATH` type or similar. Here's an [example]( https://github.com/conda-forge/gflags-feedstock/blob/2081a153339100acbb0d7ee6781ebf0c6ab1be6e/recipe/bld.bat#L7-L13 ).